FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v7.6.1
```

Comment:
7.7.0 - Every milestone is now a cross-Firebase minor release update. Only emergency bug fixes are patches.

Reply:
done